### DISCUSSION TITLE
Reverse Alias Replacement fail DKIM check?

### DISCUSSION THREAD

**Discussion by nokernel** [simple-login/app#883] [Q&A] [answered]
Hello, I am testing out simple-login.

With custom domain all setup with SPF, DKIM, DMARC.

I enabled `Reverse Alias Replacement`, when I reply to an email I see that the recipient of this email fails the DKIM check.

Is that a normal behavior?

This is the test :

from gmail ---> simple login ---> proton
reply proton --> simple login ( Reverse Alias Replacement ) --> gmail ( fail dkim )

**Comment by nokernel** [comment, marked as answer]
Okay I got my answer, just a little search.

**Reply by vkd** [reply]
Hi @nokernel , I also cought the same behavior and I'm still curious - is that a normal behavior by design or is it possible to fix it somehow?
Could you please share some details here about your findings?
I'd appreciate for your answer.

**Comment by DNCD** [comment]
Hi @nokernel, I noticed the same behavior and am curious if it's normal or fixable. Can you share your findings? Thanks!
